wyvern/feature_store/feature_server.py

```diff
@@ -456,6 +460,140 @@ async def get_historical_features(
             results=final_df.to_dict(orient="records"),
         )
 
+    @app.post(f"{settings.WYVERN_HISTORICAL_FEATURES_PATH}_v2")
```

Comment:
Should the version here follow the standard `api/v2/get_historical_features`

Reply:
yes we can do `f"v2/{settings.WYVERN_HISTORICAL_FEATURES_PATH}"`

Also:
this API path is just a placeholder. eventually it should replace the original API.

And this service is the feature store service, not the pipeline service. so it doesn't follow the `api/vX/` pattern